examples/src/main/scala/example/akkahttp/AuthExampleApp.scala

```diff
@@ -4,13 +4,13 @@ import akka.actor.ActorSystem
 import akka.http.scaladsl.Http
 import akka.http.scaladsl.server.Directives.{ getFromResource, path, _ }
 import caliban._
-import caliban.interop.tapir.RequestInterceptor
+import caliban.interop.tapir.HttpAdapter
```

Comment:
Question: We have some overloading of `*Adapter` now, we have `TapirAdapter` + `<webstack>Adapter` and now we have `HttpAdapter`, it seems like it might be slightly confusing.

Reply:
I agree. I'd rather keep `Adapter` for what it is currently to avoid more changes, how about `HttpInterpreter`, etc? It is effectively a wrapper around the initial `interpreter` with the protocol added on top.

Reply:
Renamed